synapse/models/files.py

```diff
@@ -699,6 +702,42 @@ def getModelDefs(self):
                         'doc': 'The file offset to the beginning of the section'}),
                 )),
 
+                ('file:mime:lnk', {}, (
+                    ('flags', ('int', {}), {
+                        'doc': 'The flags specified by the LNK header that control the structure of the LNK file.'}),
+                    ('entry:primary', ('file:path', {}), {
+                        'doc': 'The primary file path contained within the FileEntry structure of the LNK file.'}),
+                    ('entry:secondary', ('file:path', {}), {
+                        'doc': 'The secondary file path contained within the FileEntry structure of the LNK file.'}),
+                    ('entry:extended', ('file:path', {}), {
+                        'doc': 'The extended file path contained within the extended FileEntry structure of the LNK file.'}),
+                    ('entry:localized', ('file:path', {}), {
+                        'doc': 'The localized file path contained within the extended FileEntry structure of the LNK file.'}),
+                    ('entry:icon', ('file:path', {}), {
+                        'doc': 'The icon file path contained within the StringData structure of the LNK file.'}),
+                    ('environment:path', ('file:path', {}), {
+                        'doc': 'The target file path contained within the EnvironmentVariableDataBlock structure of the LNK file.'}),
+                    ('environment:icon', ('file:path', {}), {
+                        'doc': 'The icon file path contained within the IconEnvironmentDataBlock structure of the LNK file.'}),
+                    ('working', ('file:path', {}), {
+                        'doc': 'The working directory used when activating the link target.'}),
+                    ('relative', ('str', {}), {
```

Comment:
Do we need to strip whitespace off this value?

Reply:
We can. It's more supposed to be a filepath, but since it can start with stuff like `..\..` and we remove that, it needs to stay a string since this is relative to the working directory (well, sometimes).

But yes, I'll add that